src/lib/data/near.js

```diff
@@ -111,33 +172,42 @@ async function accountState(near, accountId) {
 }
 
 async function sendTransactions(near, functionCalls) {
+  if (functionCalls.length == 1) {
+    const { contractName, methodName, args, gas, deposit } = functionCalls[0];
+    //if (deposit == Big(0)) {
```

Comment:
This check needs to be in place before merging, and also include the gas from the original transaction